deploy/helm/sumologic/values.yaml

```diff
@@ -4136,6 +4136,7 @@ metadata:
           json_logs:
             add_timestamp: true
             timestamp_key: timestamp
+            flatten_body: true
```

Comment:
systemd is key value pair, so this option is safe for fluentd. It's necessary for otel, as it sends body as key-value pair. This body would be put inside `log` key without this option